.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto eol=lf
```

Comment:
same here

Reply:
Similar to the other file, this file tells Git: I don't care what the user's gitconfig says, this repository uses LF line endings and they will be enforced, so check it out any files that Git determines to be text (not binary like images) that way. This is a workaround to users who will set autoCRLF in Git and end up with CRLF checked out on their Windows machines, and end up with linter errors at the end of every line.